### ISSUE TITLE
Nginx microcaching

### ISSUE THREAD

**Issue by swalkinshaw** [pantheon-systems/wordpress-at-scale#28]
I think it would be useful to add a note about Nginx microcaching to the caching page. Although it might go into a level of detail you don't want yet?

If you're interested, I can do a PR for it. It's pretty useful as a "first line of defense" especially since it mostly avoids 2 of the challenges listed. It has less complexity (if you're already using Nginx) and you don't need to worry about cache expiry since the TTL is so slow (5-10 seconds for example).

More details on it: https://www.nginx.com/blog/benefits-of-microcaching-nginx/

Trellis also [supports](https://roots.io/trellis/docs/fastcgi-caching/) it out of the box.

**Comment by joshkoenig** [comment]
Yeah, this is using nginx as a reverse-proxy with a super-short TTL. I'd be happy to include a mention of this tactic.
